src/main/java/propertyFiles/MimeTypeDisplay.properties

```diff
@@ -57,13 +71,18 @@ application/x-spss-sps=SPSS Script Syntax
 text/x-sas-syntax=SAS Syntax
 application/x-sas-syntax=SAS Syntax
 type/x-r-syntax=R Syntax
+application/vnd.wolfram.mathematica.package=Mathematica Package
+application/vnd.wolfram.mathematica=Mathematica Package
```

Comment:
These would also include other code files like Mathematica notebooks. 
Maybe another name to call them: `Wolfram Mathematica Code`

Reply:
Done.